### DISCUSSION TITLE
Move entity "Last updated" to the attributes section?

### DISCUSSION THREAD

**Discussion by deviantintegral** [home-assistant/frontend#13687] [Dashboard]
Following up from https://github.com/home-assistant/home-assistant.io/pull/23900

<img width="584" alt="entity card with last updated tooltip" src="https://user-images.githubusercontent.com/255023/189498285-03b9d0d7-635a-46b8-b402-10ffa1f8887a.png">

As a user who doesn't have knowledge of the internals of Home Assistant, I think the most common interpretation would be:

1. Last changed: The state value last changed at that time.
2. Last updated: The sensor last reported an _unchanged_ state value.

In practice, "Last updated" represents a timestamp when the state value didn't change, but [entity attributes did](https://www.home-assistant.io/docs/configuration/state_object/) (it's also updated when the value changes). However, many entities don't define any attributes but still show the last updated timestamp in the UI.

Even if they do define attributes, the information hierarchy seems off. If the last updated timestamp is related to attributes, it should be shown inside the attribute section. This is especially confusing for sensors showing history graphs with the extra height between the date and the attributes section:

<img width="592" alt="image" src="https://user-images.githubusercontent.com/255023/189536121-f2eaf0a2-0cc4-4c42-8cba-7dc4d9d176ea.png">

Could we move the last updated date into the attributes section? Then it's hidden if an entity has no attributes, and it's clear what drives it updating.

This was originally added to the tooltip in https://github.com/home-assistant/frontend/pull/6926

**Comment by ildar170975** [comment]
> Could we move the last updated date into the attributes section? 

1. I would advise to remove the tooltip and add a "last updated" line to another place **to make it always visible**.

2. One possible variant is to display the "last-updated" on the "Attributes" header:
![изображение](https://user-images.githubusercontent.com/71872483/190227107-c0a1157a-ea76-48b3-91cc-427cab07bfa3.png)

Another variant is to show it below attributes:
![изображение](https://user-images.githubusercontent.com/71872483/190227044-d15514d9-b373-4a33-8ee5-b874e3bc3cac.png)
But here the "last-updated" value is only displayed when the "Attributes" section is expanded.

Also, some entities have "more-info" with attributes always displayed:
![изображение](https://user-images.githubusercontent.com/71872483/190227440-ca89be30-18c0-4cd9-b911-cd864f3c7f08.png)

Then a possible variant may be displaying the "last_updated" below the whole "Attributes" section:
![изображение](https://user-images.githubusercontent.com/71872483/190228402-5b798673-ce04-4201-87f5-e04ad438402c.png)
![изображение](https://user-images.githubusercontent.com/71872483/190228506-b3e6886d-b07c-4b5b-bf3a-02c94d43fe25.png)

**Reply by frenck** [reply]
These are not attributes. But properties of the state object. 

They are documented and the last update is shown on the state.

**Reply by ildar170975** [reply]
> But properties of the state object.

But anyway, I suggest to always show the "last-updated" value in the "more-info" window and proposed some variants.
